### PR TITLE
Added EventLogger to log events that can be used for metrics and alarms.

### PR DESCRIPTION
**Pull request by przemos**

### CODE REVIEW

File: webapp/src/main/java/uk/gov/dvsa/motr/eventlog/Event.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package uk.gov.dvsa.motr.eventlog;
+
+
```
Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/eventlog/Event.java]: Extra line

File: webapp/src/main/java/uk/gov/dvsa/motr/web/eventlog/AccessEvent.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        return params;
+    }
+
+    @Override
+    public String getCode() {
+        return "ACCESS";
+    }
+
+    public AccessEvent setRequestMethod(String method) {
+        params.put("request-method", method);
```
Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/eventlog/AccessEvent.java]: Some methods seem to have the empty line at the start of the method and others do not.  For consistency could we have them all the same?

File: webapp/src/main/java/uk/gov/dvsa/motr/eventlog/Event.java (hunk 1 above)
Comment (przemos) [inline webapp/src/main/java/uk/gov/dvsa/motr/eventlog/Event.java]: done

File: webapp/src/main/java/uk/gov/dvsa/motr/web/eventlog/AccessEvent.java (hunk 2 above)
Comment (przemos) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/eventlog/AccessEvent.java]: done

**Pull request merged**